zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -309,6 +309,28 @@ public void testAutoRestartInterpreterAfterSchedule() throws InterruptedExceptio
   }
 
   @Test
+  public void testExportAndImportNote() throws IOException, CloneNotSupportedException,
+          InterruptedException {
+    Note note = notebook.createNote();
+    note.getNoteReplLoader().setInterpreters(factory.getDefaultInterpreterSettingList());
+
+    final Paragraph p = note.addParagraph();
+    String simpleText = "hello world";
+    p.setText(simpleText);
```

Comment:
How about make `p.getReturn()` returns non InterpreterResult type object here? Then this test can simulate exception when doing `InterpreterResult result = gson.fromJson(resultJson, InterpreterResult.class);`

I think easiest way to set arbitrary object to [Job.result](https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-interpreter/src/main/java/org/apache/zeppelin/scheduler/Job.java#L75) is making [Job.setResult()](https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-interpreter/src/main/java/org/apache/zeppelin/scheduler/Job.java#L258) public.